src/inspector_agent.cc

```diff
@@ -636,6 +680,11 @@ void AgentImpl::WorkerRunIO() {
   err = uv_async_init(&child_loop_, &io_thread_req_, AgentImpl::WriteCbIO);
   CHECK_EQ(err, 0);
   io_thread_req_.data = this;
+  if (!script_name_.empty()) {
+    uv_fs_t req;
+    if (!uv_fs_realpath(&child_loop_, &req, script_name_.c_str(), nullptr))
+      script_path_ = reinterpret_cast<char*>(req.ptr);
```

Comment:
Maybe write this as `script_path_ = std::string(reinterpret_cast<char*>(req.ptr));` - without further context, the reader could be mistaken thinking it's assigning a pointer here.

You need to call `uv_fs_req_cleanup()` afterwards, by the way.

(Also, for bonus points: write the condition as `0 == uv_fs_realpath(...)`.)